src/main/webapp/style.scss

```diff
@@ -178,6 +188,19 @@ body {
   margin: 15px;
 }
 
+.clear-log {
+  position: absolute;
+  bottom: 0;
+}
+
 input {
   padding: 5px;
 }
+
+.deletedtoken {
+  color: red;
+}
+
+.addedtoken {
+  color: green;
```

Comment:
nit: consider adding green to the root & using it (also can change the name of the variable in root for red and use it in deletedtoken